.github/actions/install-dependencies/action.yml

```diff
@@ -1,12 +1,31 @@
 name: 'Install dependencies'
 description: 'Install dependencies for building and publishing containers'
+
+inputs:
+  tag:
+    description: Built container tag.
+    required: false
+    default: ""
+
 runs:
   using: "composite"
   steps:
-  - name: Install packages
+  - name: Install python packages with older ansible for Centos 8
+    if: ${{ inputs.tag == 'centos-8' }}
+    shell: bash
+    run: |
+      sudo pip3 install ansible==9.8 passlib
+
+  - name: Install python packages
+    if: ${{ inputs.tag != 'centos-8' }}
     shell: bash
     run: |
       sudo pip3 install ansible passlib
+
+  - name: Install deb packages
+    if: ${{ inputs.tag != 'centos-8' }}
```

Comment:
Why? podman and docker-compose must be installed.

Reply:
fixed